### PR TITLE
fix: separate string to view code/project

### LINKED ISSUE #47871: Showing both options as view in selection menu while claiming certificate
### Describe the Issue

It looks like there is a typo on this page.

### Affected Page

https://www.freecodecamp.org/settings#certification-settings

### Expected behavior

It should show the view code and view as website.

### Screenshots
Screenshot from 2022-10-06 18-42-40

### System

- Device: [Laptop]
- OS: [Zorin OS]
- Browser: [edge]
- Version: [106]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Prince-Mendiratta**
Signed-off-by: Prince Mendiratta <prince.mendi@gmail.com>

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47871 

To be more descriptive on what does the View Button does for Multi file project solutions, separate text for viewing Code and viewing Project.

### CODE REVIEW

**Review by Sboonny — COMMENTED** [review]
Because `View` exist as innerText for the dropdown, I think we can move away from adding it, because the dropdown will act as indicator what the buttons do (View --> Code).

File: client/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
     "frontend": "Front End",
     "backend": "Back End",
     "view": "View",
+    "view-code": "View Code",
+    "view-project": "View Project",
```
Comment (Sboonny) [inline client/i18n/locales/english/translations.json:13]:
```suggestion
    "view-code": "Code",
    "view-project": "Project",
```

Comment (Prince-Mendiratta) [reply client/i18n/locales/english/translations.json:13]: Makes sense, will change and add the preview icon as well.

**Review by raisedadead — COMMENTED** [review]
We will need some tests to go with this because this has a larger-than-normal blast radius.

**Comment by Prince-Mendiratta** [comment]
@raisedadead How can I proceed with adding a test for checking the blast radius? Is there any other implementation for the same I can take as a reference?

File: client/i18n/locales/english/translations.json (hunk 1 above)
Comment (ieahleen) [reply client/i18n/locales/english/translations.json]: isn't this going to make localization harder?

**Comment by raisedadead** [comment]
See https://contribute.freecodecamp.org/#/how-to-add-cypress-tests and for more help reach out to the chat or the forum.

**Comment by Sboonny** [comment]
I think it's better to use Jest to map how much this effect, and if something like this happens again it notifies reviewers, oh this component is affecting these multiple pages, please check them as well.

Something like [map.test.tsx](https://github.com/freeCodeCamp/freeCodeCamp/blob/a0ffc91aa06acb95149ca7f4a67a77f3cb55be46/client/src/components/Map/map.test.tsx), this should be [jestjs](https://jestjs.io/)

I can't find the documents about jest, image. if there is no document, I should open issue for that as well 🤔 😓 


@Prince-Mendiratta Unfortunately Jest tests are added before my time in freeCodeCamp, so I can't help you much.

**Comment by raisedadead** [comment]
Ah - right. 

I wasn't thinking straight. 

Yes - actually we just need unit tests (Jest) not integration tests (Cypress). They are pretty straightforward to add/update by referencing the files @Sboonny shared.

File: client/i18n/locales/english/translations.json (hunk 1 above)
Comment (raisedadead) [reply client/i18n/locales/english/translations.json]: @ieahleen is correct these should be whole phrases, even at the cost of repeating ourselves. Assuming you can combine words is a bad idea. 

We should go back to adding individual keys for each label and keep it straight.

Comment (Prince-Mendiratta) [reply client/i18n/locales/english/translations.json]: I'll remove the previous commit and force push in that case.

Comment (Sboonny) [reply client/i18n/locales/english/translations.json]: Thank you and sorry for the confusion

**Comment by Prince-Mendiratta** [comment]
Hi, have added a test for checking for the existence of the proper text for any component displaying both Project preview and Source Files.
image


A thing i noticed though is that in the file [certification.test.tsx](https://github.com/freeCodeCamp/freeCodeCamp/blob/977911bf84dad2f21dc7d46688e04e7ca2c19288/client/src/components/settings/certification.test.tsx#L252) uses a single object for all the various test cases. I believe this can be improved by having separate object props for each different category of View Button that has to be displayed, will make it more intuitive and easier to make additions to. If you're interested, I could open a new issue and try this out.

**Comment by raisedadead** [comment]
> If you're interested, I could open a new issue and try this out.

Yes - It's OK to refactor. Go ahead and create PR directly. I think it doesn't warrant an issue of its own - which is what we do when we need to have more discussion before thinking about a PR.

**Comment by raisedadead** [comment]
Can you lint and format the files? Scripts are available in the docs and the package.json - Thanks

**Comment by Prince-Mendiratta** [comment]
woops

**Review by raisedadead — APPROVED** [review]

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @Prince-Mendiratta, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Pull request merged**